### PR TITLE
Feature/6 hiring staff can edit vacancies after they are published

### PR DESCRIPTION
**Pull request by despo**

### CODE REVIEW

**Comment by despo** [comment]
[Hiring staff can edit vacancies after they are published](https://trello.com/c/fr6Emb5D/6-hiring-staff-can-edit-vacancies-after-they-are-published)

File: app/views/hiring_staff/vacancies/edit.html.haml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+%h1.heading-large
+  = t('vacancies.edit_heading', school: @vacancy.school.name)
+
```
Comment (gazaston) [inline app/views/hiring_staff/vacancies/edit.html.haml]: Do we need some way for the user to cancel and return to the `schools#show` page?

File: app/views/hiring_staff/schools/_vacancy.html.haml [hunk 2]
```diff
@@ -3,5 +3,6 @@
   %td= format_date(vacancy.publish_on)
   %td= format_date(vacancy.expires_on)
   %td= vacancy.status.titleize
+  %td= link_to 'Edit', edit_school_vacancy_path(school, vacancy.id)
```
Comment (gazaston) [inline app/views/hiring_staff/schools/_vacancy.html.haml]: We'll need to add a `colspan=2` attribute to the 'Actions' table heading cell — the border cuts off prematurely.

screen shot 2018-04-03 at 10 37 32

File: app/views/hiring_staff/vacancies/edit.html.haml (hunk 1 above)
Comment (despo) [reply app/views/hiring_staff/vacancies/edit.html.haml]: @gazaston yes. Do you have any suggestions?

Comment (gazaston) [reply app/views/hiring_staff/vacancies/edit.html.haml]: Probably just a simple link above the `h1`?

Comment (despo) [reply app/views/hiring_staff/vacancies/edit.html.haml]: @gazaston done! After this PR is in we should maybe address navigation more generally. E.g. review page -> school page, vacancy#show -> school page etc.

Comment (gazaston) [reply app/views/hiring_staff/vacancies/edit.html.haml]: I'll merge this in as-is, but agree that we need to tackle this nav pattern. I'll address that immediately.

**Pull request merged**